ext/rbs_extension/parser.c

```diff
@@ -963,25 +963,25 @@ static VALUE parse_simple(parserstate *state) {
     return type;
   }
   case kBOOL:
-    return rbs_base_type(RBS_Types_Bases_Bool, rbs_location_current_token(state));
+    return rbs_bases_bool(rbs_location_current_token(state));
```

Comment:
These are just working names for now, we'll reconsider these once we decide what the new public C API looks like.